### PR TITLE
Drinking Alcohol Now Makes You Drunk

### PR DESCRIPTION
**Pull request by ThatDan123**
Adds the visual effect for being drunk.

Currently set to add how much you drink to the time, eg 5u = 5 seconds. 

However this happens for each type of alcohol, so the more types of alcohol in a drink the more drunk you get from each sip.

### CODE REVIEW

**Comment by ThatDan123** [comment]
Ready for a review, tested on headless, two clients and was fine.

File: UnityProject/Assets/Scripts/Food/DrinkableContainer.cs [hunk 1]
```diff
@@ -71,11 +72,31 @@ private void Drink(PlayerScript eater, PlayerScript feeder)
...
+		if ((int) drinkAmount == 0) return;
+
+		foreach (var reagent in container.CurrentReagentMixGet)
+		{
+			//if its not alcoholic skip
+			if (!AlcoholicDrinksSOScript.Instance.AlcoholicReagents.Contains(reagent.Key)) continue;
+
+			//The more different types of alcohol in a drink the longer you get drunk for each sip.
+			playerEatDrinkEffects.ServerSendMessageToClient(eater.gameObject, (int)drinkAmount);
+			playerEatDrinkEffects.ServerSendMessageToClient(eater.gameObject, 0);
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Food/DrinkableContainer.cs]: why does it send it then send 0?

File: UnityProject/Assets/Scripts/Camera/CameraEffectControlScript.cs [hunk 2]
```diff
@@ -27,5 +32,24 @@ public void ToggleNightVisionEffectState()
 		{
 			nightVisionCamera.enabled = !nightVisionCamera.enabled;
 		}
+
+		private void Update()
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Camera/CameraEffectControlScript.cs]: maybe better served to use the update manager?

File: UnityProject/Assets/Scripts/Food/DrinkableContainer.cs (hunk 1 above)
Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Food/DrinkableContainer.cs]: Ah, good catch, thats was when i was testing with sync vars.

File: UnityProject/Assets/Scripts/Camera/CameraEffectControlScript.cs (hunk 2 above)
Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Camera/CameraEffectControlScript.cs]: changed.

**Pull request merged**